src/__experimental__/components/decimal/decimal.stories.mdx

```diff
@@ -268,3 +268,27 @@ For more information check our [Validations](?path=/docs/documentation-validatio
 Due to the `Textbox` component being used internally by the `Decimal` component most of the `Textbox` props are appliable to `Decimal`
 
 <Props of={OriginalTextbox} />
+
+### Custom locale
+
+<Preview>
+  <Story name="custom locale">
+    {() => {
+      const [state, setState] = useState("0.01");
+      const setValue = ({ target }) => {
+        setState(target.value.rawValue);
+      };
+      const [statePt, setStatePt] = useState("0.01");
+      const setValuePt = ({ target }) => {
+        setStatePt(target.value.rawValue);
+      };
+      return <>
+        <Decimal label="Decimal - pt locale" value={statePt} onChange={setValuePt} locale="pt" delimiter=" " separator="," />
```

Comment:
Why do we need to pass in `locale` AND `delimiter`/`separator`. If we are being told the delimiter and the separator then we shouldn't need the locale (or vice versa)

Reply:
I used the same approach as in code example: https://codesandbox.io/s/numberformatting-kijbl?file=/src/locale.js - we can change it to use delimiter and separator only.

Reply:
I think i would prefer that we expect delimiter and separator to be passed in. Then the consuming application can use whatever library they want for I18n.